editor.planx.uk/src/@planx/components/Checklist/Editor.tsx

```diff
@@ -295,14 +292,14 @@ export const ChecklistComponent: React.FC<ChecklistProps> = (props) => {
               ...values,
               ...(groupedOptions
                 ? {
-                    categories: groupedOptions.map((gr) => ({
-                      title: gr.title,
-                      count: gr.children.length,
-                    })),
-                  }
+                  categories: groupedOptions.map((gr) => ({
```

Comment:
nit: what do you think about changing these acronyms (`gr`) to something a bit more meaningful? 

I just find it slightly increases the mental load unnecessarily to read something that isn't immediately obvious (like I understood it was a group on second reading, but I think making code as frictionless and easy to read as possible is something to strive for!)

Reply:
Fully agree - updated now !